browser/main/modals/PreferencesModal/UiTab.js

```diff
@@ -569,6 +574,24 @@ class UiTab extends React.Component {
               />
             </div>
           </div>
+          <div styleName='group-checkBoxSection'>
+            <label>
+              <input onChange={(e) => this.handleUIChange(e)}
+                checked={config.preview.allowCustomCSS}
+                ref='previewAllowCustomCSS'
+                type='checkbox'
+              />&nbsp;
+              {i18n.__('Allow custom CSS for preview')}
+            </label>
+          </div>
+          <div styleName='group-section'>
+            <div styleName='group-section-label'>
+              {i18n.__('Custom CSS')}
+            </div>
+            <div styleName='group-section-control'>
+              <ReactCodeMirror onChange={e => this.handleUIChange(e)} ref={e => (this.customCSSCM = e)} value={config.preview.customCSS} options={{ lineNumbers: true, mode: 'css', theme: codemirrorTheme }} />
```

Comment:
This `ReactCodeMirror` is too huge...
Set the width of  this according to [the editor](https://github.com/BoostIO/Boostnote/blob/master/browser/main/modals/PreferencesModal/UiTab.js#L260).
<img width="2545" alt="2018-06-01 14 32 15" src="https://user-images.githubusercontent.com/14838850/40823055-d4c505a8-65a9-11e8-8f54-2b7bb8d06d73.png">